src/libtriton/engines/symbolic/pathManager.cpp

```diff
@@ -231,7 +231,7 @@ namespace triton {
 
 
       /* Pushes constraint created from node to the current path predicate. */
-      void PathManager::pushPathConstraint(const triton::ast::SharedAbstractNode& node) {
+      void PathManager::pushPathConstraint(const triton::ast::SharedAbstractNode& node, const std::string& comment="") {
```

Comment:
As a matter of homogeneous coding style, can you define the default parameter in `.hpp` and not `.cpp`?